.gitignore

```diff
@@ -4,3 +4,5 @@ node_modules/
 out/
 raw/
 .history
+!tests/packages/**/*/node_modules/
+!tests/packages/node_modules/
```

Comment:
With the entries in ` tests/packages/.gitignore`, are these lines necessary as well?